pkg/tfbridge/diff.go

```diff
@@ -309,9 +309,71 @@ func makeDetailedDiff(
 		makePropertyDiff(ctx, en, string(k), v, tfDiff, diff, forceDiff, etf, eps, true, useRawNames(etf))
 	}
 
+	// In some cases, the terraform diff will have additional attributes not covered by Pulumi's olds and news.
+	// We want to make sure that we do not ignore these changes, if any.
+	// In order to look up whether our attributes have been added to the Pulumi `diff`, we first need to parse the
+	// Terraform diff paths (of the format `foo.0.bars.1.baz[...]`) to Pulumi (of the format `foo[0].bar[1].baz[...]`.
+	for path := range tfDiff.Attributes() {
+
+		pulumiPath, specificDiff := parseTFPath(path, tfs, ps)
+		// Look up the resulting path and register any diffs if not present in Pulumi diff.
+		if _, ok := diff[pulumiPath]; !ok {
+			if specificDiff {
+				// TODO: Using PropertyDiff_ADD is not entirely accurate in every case.
+				// TODO: (cont) We may want to pass on a more generic UPDATE strategy here - less precise, less error prone.
```

Comment:
Can you give an example for the case there `PropertyDiff_ADD` is the wrong case? What is the effect of getting the kind wrong (is it just display)?

Reply:
It could be a PropertyDiff_UPDATE, on an already-existing object key.